signup-service/src/main/java/io/mosip/signup/services/RegistrationService.java

```diff
@@ -101,6 +101,9 @@ public class RegistrationService {
     @Value("${mosip.signup.challenge.resend-attempt}")
     private int resendAttempts;
 
+    @Value("${mosip.signup.challenge.verification-attempt}")
```

Comment:
Please raise a pr to mosip config adding this change to both camdec-dev, and release-1.4.1-ES branches.

Reply:
PR of camdgc-dev https://github.com/mosip/mosip-config/pull/5990

Reply:
PR of config for camdgc-dev